Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseTeleOp.java

```diff
@@ -53,18 +52,14 @@ public void TeleOpDrive () {
 
         //Driving / pivoting, not sure how we want it to be structured so this can be fixed later
         if(Math.atan2(FilteredInput[0], FilteredInput[1]) < 20){
```

Comment:
Also what is so special about this number 20? Please make it a constant since you're using it in multiple places and we don't know what it means. https://en.wikipedia.org/wiki/Magic_number_(programming)